### PR TITLE
using directive to flatten everything

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/_help.py [hunk 1]
```diff
@@ -188,57 +107,26 @@
...
-        text: |-
-               az managed-network managed-network-peering-policies show --managed-network-name \\
-               "myManagedNetwork" --managed-network-peering-policy-name "myHubAndSpoke" --resource-group \\
-               "myResourceGroup"
 """
 
 helps['managed-network managed-network-peering-policies create'] = """
     type: command
     short-summary: The Put ManagedNetworkPeeringPolicies operation creates/updates a new Managed Network Peering Policy
-    examples:
```
Comment (zikalino) [inline src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/_help.py]: seems all examples disappeared, I assume they will require additional fix?

Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/_help.py]: yes, Changlong is working on it

File: README.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ use-extension:
   #"@autorest/python": "latest"
   "@autorest/python": "5.0.0-dev.20200211.1"
 
+python: true
+cli: true
+clicommon: true
+require: 
+  - readme.python.md
```
Comment (zikalino) [inline README.md]: doesn't require any $(thisfolder) or something like that in the front?

Comment (qiaozha) [reply README.md]: no

**Pull request merged**